docs/examples/webpack/babel.config.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Babel config
+ *
+ * @type {import('@babel/core').ConfigFunction}
+ */
+module.exports = function (api) {
+  const browserslistEnv = !api.env('test')
+    ? 'production'
```

Comment:
Just a note to say you'd typically see projects use `development` to run fewer transforms locally (for example, targeting modern browsers only) but we're using `production` here by default

Until we have an [environment named `production`](https://github.com/browserslist/browserslist#configuring-for-different-environments) this will return our default browsers in **.browserslistrc**